src/pal/pal_net_posix.c

```diff
@@ -1129,6 +1129,9 @@ int32_t pal_ntop(
     prx_size_t addr_string_size
 )
 {
+#ifndef NI_MAXSERV
+#  define NI_MAXSERV 32 // use same definition as in os_mock.h if not defined
```

Comment:
I have just seen, that /usr/include/netdb.h contains the following:
`# ifdef __USE_MISC`
`#  define NI_MAXHOST      1025`
`#  define NI_MAXSERV      32`
`# endif`
I had created commit 963a5a1cebbf3799bd4c42c9f60b3af06c55a680 before 28ffd768bca96951013dc203fbe3d1958c8d6f33 .
So the modification above may be obsolete or unnecessary.